### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by lorenzoworx**
## Learning objectives

- Use basic JavaScript syntax.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Understand how to parse a Figma design to create a UI.
- Use objects to store and access data.

### CODE REVIEW

File: portfolio.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    gridClass: ['grid3', 'grid4', 'grid5', 'grid6', 'grid7', 'grid8'],
+    heading: 'Professional Art <br> Printing Data',
+    h1Class: 'crete w400 s32 lh44 white p16',
+    textContent: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    textClass: 'inter w400 s16 lh24 white p16',
+    ulClass: 'techlist p16',
+    tags: ['css', 'html', 'bootstrap', 'Ruby'],
+    tagClass: 'inter w500 s16 lh20 white project2Tag',
+    buttonText: 'See Project',
+  },
```
Comment (brainconnect93) [inline portfolio.js:32]: - [x] Great effort working on the pop-up modal of your project 👏 👏 . But I would suggest we stick to the project requirement as much as possible, From the project in your `Most recent work`, you have 7 cards but currently, in the `desktop version` you have only created two cards for pop-up functionality the remaining cards were not functioning well. Please kindly fix this by making all the cards have a pop-up modal functionality. 👍 👍 
> N.B: The `mobile version` of the pop-up modal for all the cards works just fine.

File: styles.css [hunk 2]
```diff
@@ -364,6 +373,77 @@ a.logo {
...
+  height: 100%;
+  position: fixed;
+}
+
+#popup {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  padding: 16px;
+}
```
Comment (brainconnect93) [inline styles.css:397]: - [x] Having made a great effort, I must commend you that you have done wonderfully well, 👍 👍  but I noticed your pop-up card in the `mobile version` are not well aligned, the card is hitting the wall to the bottom. I would kindly give this card some `margin bottom`.

- [x] Secondly, I noticed you gave the cards container a `background color` that affects the main page from showing as seen in the Figma design below. When the pop-up modal card is displayed, the main page should still be showing underneath but in a Blur manner. You can do this by giving your card container a CSS property called `backdrop filter` with a value of `blur(8px)

> The card display well on the `desktop screen`......Kudos 👍 👍 

Your mobile Pop-up design 👇 

lor-pp

Figma design 👇 

pp2-mob

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :recycle:

### Hi @lorenzoworx, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.js (hunk 1 above)
Comment (lorenzoworx) [reply portfolio.js:32]: Good morning. Please on the Figma design file, there aren't any 'see project' buttons, so where exactly should I attach the event listeners?

**Review by TadesseDev — APPROVED** [review]
### Hi @lorenzoworx 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
ST3D


## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅
 - [ ]  Correct GitHub Flow. ✅
 - [ ]  Good commit history. ✅
 - [ ]  Good PR title/description. ✅
 - [ ]  All linters passed ✅
## Optional suggestions
- [Optional] Dear @lorenzoworx, Impressive changes. One thing I notice is when I switch from desktop to mobile view the style and layout for the modal remains as it was `Desktop version`) I would be great to have some way of switching the modal from mobile to desktop and vise versal without having to reload. 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**